### PR TITLE
scaffolder/next: simplify the exported components

### PR DESCRIPTION
**Pull request by benjdlambert**
- chore: moving things around a little
- chore: fix api reports

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by benjdlambert** [comment]
Going to piggyback on the changeset from #16528 as that's not been released yet, and this PR is basically just fixing some things I noticed post merge that we can make simpler.

File: plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx [hunk 1]
```diff
@@ -132,15 +133,12 @@ export const OngoingTask = (props: {
         ) : null}
 
         <Box paddingBottom={2}>
-          <Paper style={{ position: 'relative', overflow: 'hidden' }}>
-            <TaskBorder
```
Comment (benjdlambert) [inline plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx:136]: Thought that this component was pretty useless on it's own, so wrapped it all up in the paper instead and just export that @dagda1

Comment (benjdlambert) [reply plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx:136]: Or do you think it would be useful to be able to access just the steps on their own?

Comment (dagda1) [reply plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx:136]: @benjdlambert I think this makes sense.  Having the two components separate is a nothing gain and maybe more confusing

**Review by Rugvip — APPROVED** [review]
:shipit: !

**Pull request merged**